src/components/SecretsModal/SecretsModal.scss

```diff
@@ -14,10 +14,6 @@ limitations under the License.
 @import '~carbon-components/scss/globals/scss/vars';
 
 .modal.bx--modal.is-visible {
-  .bx--modal-container {
-    width: 30%;
-  }
-
   .bx--form-item,
```

Comment:
I already showed you this working but for sake of book keeping.
Instead of importing the `KeyValues` css file here to overwrite the margin for the `.bx--form-item`, it is less of hassle if you change this style to: 
``` css
.bx--modal-content > form > .bx--form-item,
  .bx--dropdown__wrapper {
    margin-bottom: $spacing-03;
  }
```
With this style the only `.bx--form-item` elements that will be affected will only be those first order child elements inside the form tag. Meaning that the ones inside `keyvalues` div will not be changed.